buildlib/tools/coverity.sh

```diff
@@ -6,14 +6,31 @@ source ${reldir}/common.sh
 # Run Coverity and report errors
 # The argument is a UCX build type: devel or release
 #
+
+modules_for_coverity() {
+	res=0
+	module_load $COV_MODULE
+	res=$(($res+$?))
+	module_load $CUDA_MODULE
+	res=$(($res+$?))
+	module_load $GDRCOPY_MODULE
+	res=$(($res+$?))
+	module_load $JDK_MODULE
+	res=$(($res+$?))
+	module_load $MVN_MODULE
+	res=$(($res+$?))
+	module_load $XPMEM_MODULE
+	res=$(($res+$?))
+	return $res
+}
+
 run_coverity() {
-	echo 1..1 > coverity.tap
 
 	modules_for_coverity
 
-    if [ $res -gt 0 ]; then
-    	azure_log_issue "All modules cannot load"
-    fi
+	if [ $res -gt 0 ]; then
+		azure_log_issue "All modules cannot load"
```

Comment:
Some module could not be loaded

Reply:
@yosefe  Why? change on azure_log_warning?

Reply:
because this error is print if one or more were not leaded right? not when all modules were not loaded

Reply:
If all modules were not loaded error will print too